server/sonar-server/src/main/java/org/sonar/server/platform/monitoring/SettingsSection.java

```diff
@@ -19,41 +19,54 @@
  */
 package org.sonar.server.platform.monitoring;
 
-import com.google.common.collect.ImmutableSortedMap;
 import java.util.Map;
-import java.util.SortedMap;
 import org.sonar.api.PropertyType;
 import org.sonar.api.config.PropertyDefinition;
 import org.sonar.api.config.PropertyDefinitions;
 import org.sonar.api.config.Settings;
+import org.sonar.api.server.ServerSide;
+import org.sonar.process.systeminfo.Global;
+import org.sonar.process.systeminfo.SystemInfoSection;
+import org.sonar.process.systeminfo.protobuf.ProtobufSystemInfo;
 
 import static org.apache.commons.lang.StringUtils.abbreviate;
+import static org.apache.commons.lang.StringUtils.containsIgnoreCase;
+import static org.apache.commons.lang.StringUtils.endsWithIgnoreCase;
+import static org.sonar.process.systeminfo.SystemInfoUtils.setAttribute;
 
-public class SettingsMonitor implements Monitor {
+@ServerSide
+public class SettingsSection implements SystemInfoSection, Global {
 
   static final int MAX_VALUE_LENGTH = 500;
+  private static final String PASSWORD_VALUE = "xxxxxxxx";
   private final Settings settings;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "Settings"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)